hosts/configuration.nix

```diff
@@ -188,9 +188,11 @@
     home = {
       stateVersion = "22.05";
     };
-
     programs = {
       home-manager.enable = true;
     };
+    imports = [
+      inputs.plasma-manager.homeManagerModules.plasma-manager
+    ];
```

Comment:
Maybe /modules/desktops/kde.nix is a better location for this, no? This import is within the scope of any host and any desktop environment at the moment, which possibly is not ideal.
Force push this update. Should be ready to merge then.